mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -63,14 +63,44 @@ def calculate_statistics_overview(self, df, is_clean=True):
 
             # Aggregated stats
             column_count = len(df.columns)
+            row_count = df.shape[0]
+
             data['total_null_value_count'] = sum(
                 data[f'{col}/null_value_count'] for col in df.columns
             )
+            data['total_null_value_rate'] = (
```

Comment:
let's create a method for calculating the rate. the logic of checking denominator is 0 is repeated.